src/utils/utils.js

```diff
@@ -4,7 +4,25 @@ export function getAllArticles() {
     return axios
         .get("https://stumpyjoe.onrender.com/api/articles")
         .then((response) => {
-            console.log(response.data.articles)
+            // console.log(response.data.articles)
             return response.data.articles
         })
+}
+
+const myApi = axios.create({
+    baseURL: 'https://stumpyjoe.onrender.com/api',
+  });
+
+export function fetchArticleById(article_id) {
+    return myApi.get(`/articles/${article_id}`).then((res) => {
+        // console.log(res.data.article)
```

Comment:
Remember to get rid of console.logs and commented out code before submitting finalised code 🗑️